src/components/GatherList/styles.ts

```diff
@@ -49,9 +65,26 @@ export const ItemDetail = styled.div`
   flex-grow: 2;
   gap: 8px;
   padding-left: 3px;
+
+  span {
+    ${gatherBreakpoints.mobile} {
+      font-size: 14px;
+    }
+  }
```

Comment:
```suggestion
    ${gatherBreakpoints.mobile} {
        span {
              font-size: 14px;
        }
    }
```

보통 이런 방식으로 사용하는 것으로 알고있습니다~